### DISCUSSION TITLE
Is it possible to integrate into NX Monorepo?

### DISCUSSION THREAD

**Discussion by collinosh** [farm-fe/farm#1466] [Q&A] [answered]
Would love to give this a try but my setup is somewhat unconventional, with a heavy reliance on the NX ecosystem.  The documentation seems to suggest it's vite compatible and more or less plug-and-play, wondering if that changes given my current configuration.

**Comment by wre232114** [comment, marked as answer]
Yes, you can integrate Farm into nx monorepo

**Reply by collinosh** [reply]
That's sick. I'll give it a try this weekend, thanks for the quick response!

**Reply by wre232114** [reply]
Free to reply here if you meet problems
